src/admin/menu/views/MenuEdit.tsx

```diff
@@ -142,22 +128,20 @@ const MenuEdit: FunctionComponent<MenuEditProps> = ({ history, match }) => {
 				},
 				update: ApolloCacheManager.clearNavElementsCache,
 			})
-				.then(() => handleSucces('Het navigatie item is succesvol geüpdatet'))
-				.catch(err => handleError('Het updaten van het navigatie item is mislukt', err));
+				.then(() => handleResponse('Het navigatie item is succesvol geüpdatet'))
+				.catch(err => handleResponse('Het updaten van het navigatie item is mislukt', err));
```

Comment:
als de functie hier een error smijt die undefined is, dan krijg je een toast:
'Het updaten van het navigatie item is mislukt' met als type: success

je kan dit oplossen door:
handleResponse('Het updaten van het navigatie item is mislukt', err || null)

en dan te checken of het undefined is => show success message
als het null is of defined => show error

Reply:
Lijkt me dan wel mediocre error handling aan de andere kant, toch? Either way, Ik van het hier ook even op.